applications/app/controllers/OptInController.scala

```diff
@@ -20,10 +20,12 @@ object OptInController extends Controller {
     feature match {
       case "https" => HTTPS.opt(choice)
       case "header" => Header.opt(choice)
+      case "gallery" => gallery.opt(choice)
       case _ => NotFound
     }
   }
 
   val HTTPS = OptInFeature("https_opt_in")
   val Header = OptInFeature("new_header_opt_in")
+  val gallery = OptInFeature("gallery_redesign_opt_in")
```

Comment:
Note: This was discussed in person and the fastly change that makes this possible has already been made in a separate PR ready to go :)